### PR TITLE
[table] replace before's drag handler by a full HTML element

### LINKED ISSUE #798: [table] replace before's drag handler by a full HTML element
As we have our handler on first td element, it's possible to drag the line by clicking inside a textfield.

### PR DESCRIPTION
**Pull request by almonais**
fixes #798 

image

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://831-pr-lucca-front-LuccaSA.surge.sh

File: packages/scss/src/components/_table.scss [hunk 1]
```diff
@@ -149,19 +149,52 @@
...
+		width: 2rem;
+		cursor: move;
+	  
+		&:hover {
+		  &::after {
+			color: _color('text.light');
+		  }
+		}
+	  
+		&::after {
```
Comment (sanlucca) [inline packages/scss/src/components/_table.scss]: You have an absolute positionned element in a absolute positionned element. It is not optimal. Isn't it possible to have the after not being positioned absolute ?

File: packages/scss/src/components/_table.scss [hunk 2]
```diff
@@ -149,19 +149,49 @@
 		.table-body-row-cell:first-child {
 			padding-left: 2rem;
 			position: relative;
+		}
+		
+		// .table-body-row-cell:first-child {
```
Comment (sanlucca) [inline packages/scss/src/components/_table.scss]: You can remove this

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://831-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-831

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-831

**Pull request merged**